src/client/app/chat/chat.component.ts

```diff
@@ -367,6 +399,14 @@ export class ChatComponent implements OnInit, AfterViewChecked {
         this.videoMessage.text = 'Video Component';
         this.socketService.sendMessage(this.videoMessage);
       });
+    /*this.videoMessage.receiverId = this.chatService.getGroup().id;
```

Comment:
createVideo(videos: FileList) {
    this.chatService.uploadFile(videos)
      .subscribe(res => {
        this.videoMessage.contentData.data = res._body;
        this.videoMessage.receiverId = this.chatService.getGroup().id;
        this.videoMessage.senderId = this.selectedUser.id;
        this.videoMessage.receiverType = 'group';
        this.videoMessage.contentType = 'video';
        this.videoMessage.type = 'video';
        this.videoMessage.status = 'delivered';
        this.videoMessage.text = 'Video Component';
        this.socketService.sendMessage(this.videoMessage);
      });
  }